### PR TITLE
feat: review

### PR DESCRIPTION
**Pull request by oOWinOo**
## What did I do

- Edit Schema for Review. 

- Create API for get review by hostID and filter
- Create API for CREATE review 
- Create API for GET review number for each score 

- Create Review page.

### CODE REVIEW

File: src/app/review/_components/ReviewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+function StarMaker(props: { rating: number }) {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="flex">
+        {Array.from({ length: 5 }).map((_, index) => {
+          return (
+            <div key={index}>
+              <FontAwesomeIcon
+                icon={faStar}
+                className={`h-4 w-4 pr-1 ${index <= props.rating - 1 ? "text-pending" : "text-neutral-200"}`}
```
Comment (OnFireByte) [inline src/app/review/_components/ReviewCard.tsx:70]: ใช้ function cn ไม่ทำ string interpolation กับ tailwind class

File: src/app/review/page.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  //       </Link>
+
+  //       <Link href="/signin">
+  //         <button className="rounded-2xl bg-slate-500/20 px-10 py-3 font-semibold transition hover:bg-slate-500/40">
+  //           Sign In
+  //         </button>
+  //       </Link>
+  //     </div>
+  //   </main>
+  // );
```
Comment (OnFireByte) [inline src/app/review/page.tsx:126]: ลบ

File: src/server/db/schema.ts [hunk 3]
```diff
@@ -74,10 +74,12 @@ export const hostRelation = relations(hostUser, ({ one, many }) => ({
 }));
 
 export const ratingAndReview = sqliteTable("rating_review", {
-  reviewID: text("review_id", {
-    length: 64,
-  }).primaryKey(),
-  appointmentID: text("appointment_id", {
+  rid: int("id", {
```
Comment (OnFireByte) [inline src/server/db/schema.ts:77]: id ไปเลย

File: src/app/review/page.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        <SelectTrigger
+          className={cn(
+            "w-full",
+            "border-none",
+            "text-placeholder",
+            "bg-neutral-100",
+            "w-[124px]",
+          )}
+
+          // className={cn("w-full", "border-none", isMobile && "bg-neutral-100")}
```
Comment (OnFireByte) [inline src/app/review/page.tsx:61]: ลบ

**Comment by OnFireByte** [comment]
รัน `git rm --cached libsql/local.db`
image

**Comment by OnFireByte** [comment]
pnpm format
image

**Review by OnFireByte — APPROVED** [review]

File: src/app/myevent/_components/Card.tsx [hunk 5]
```diff
@@ -81,6 +82,7 @@ export function Card(prop: EventProps) {
           )
         );
       case EventStatus.COMPLETED:
+        console.log("ITS COMPLETE FUCK");
```
[the remark this answers is not in the document]
Comment (oOWinOo) [reply src/app/myevent/_components/Card.tsx:85]: Ops

**Pull request merged**